cypress/integration/settings.js

```diff
@@ -0,0 +1,12 @@
+it('should be able to open and close the settings modal', () => {
+    cy.visit('/')
+        .get('[data-testid=component-name]')
+        .get('#settingsPanelOpenButton')
+        .click()
+        .get('#settingsPanel')
```

Comment:
Could we use `data-testid`'s instead? (Unless we're using the IDs for something)